packages/terra-docs/tests/wdio/docs-spec.js

```diff
@@ -0,0 +1,9 @@
+// Terra.describeViewports('Docs', ['tiny', 'medium', 'large'], () => {
```

Comment:
The tests are commented out because it seems that by setting the package to private, the package is ignored when compiling to lib so it's not in the testing site.